WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationRepository.swift

```diff
@@ -173,7 +173,7 @@ public final class ConversationRepository: ConversationRepositoryProtocol {
         }
 
         await conversationsLocalStore.storeConversation(
-            conversation,
+            conversation.toDomainModel(),
```

Comment:
preparing data for the local store by converting the API object to a Domain object.